jamaljsr/polar
# feat(mint-taro-asset): mints a taro asset action

### Description

added action buttons to mint an asset
added modal for to adjust arguments for mint asset
added appropriate rpc calls to mint asset to proxy client and server
updated network
added tests

### Steps to Test

1. Create a new network with one bitcoind node
2. Add lnd latest
3. add taro latest
4. Mine over 100 blocks.
5.Deposit btc in alice lnd node.
6. Taro > Actions > Mint.
7. Fill oput parameters and hit mint.
8. You should see a new asset in the taro node's info tab.


## Screenshots

<img width="1009" alt="Screenshot 2023-01-04 at 9 30 14 PM" src="https://user-images.githubusercontent.com/45839100/210702000-b908acaf-ec74-478e-9ff7-72b2020ae62e.png">
<img width="565" alt="Screenshot 2023-01-04 at 9 30 28 PM" src="https://user-images.githubusercontent.com/45839100/210702014-f4afb749-c21a-44f1-a18e-4e33981661ab.png">
